src/applaunch/java/org/spongepowered/common/applaunch/config/common/OptimizationCategory.java

```diff
@@ -51,6 +51,9 @@
     @Comment("Uses theosib's redstone algorithms to completely overhaul the way redstone works.")
     public final EigenRedstoneCategory eigenRedstone = new EigenRedstoneCategory();
 
+    @Setting(value = "bell-leak-fix")
+    private boolean bellLeak = true;
```

Comment:
Could you add a quick comment to explain what it does, just so people know what they are turning on/off?

Reply:
I've added a small comment explaining the issue and how this option should fix it